### PR TITLE
Code review example

### PR DESCRIPTION
**Pull request by kyrabusser**
Here's a simple PR from me, so we can all practice doing code reviews. :smile: 

Follow the instructions from https://sites.google.com/codeustudents.com/spring-2019/code-reviews and leave me lots of comments!

I like to ask for a screenshot when reviewing code that makes visible changes--can you guess what this will look like?

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -28,11 +28,17 @@
...
+    <h1>Team 10!</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
     their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!</p>
+    <h2>Our team</h2>
+    <ul class="team-names">
+     <li>Charles</li>
```
Comment (kyrabusser) [inline src/main/webapp/index.html:38]: @CharlesArday Is there anything you think we should add here?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -28,11 +28,17 @@
...
+     <li>Hannah</li>
```
Comment (kyrabusser) [inline src/main/webapp/index.html:39]: @hengelman Any thoughts?

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -28,11 +28,17 @@
...
+     <li>Stanley</li>
```
Comment (kyrabusser) [inline src/main/webapp/index.html:40]: @stanleynwakamma What do you think?

**Review by kyrabusser — COMMENTED** [review]
Hi team! Here are some examples of in-line PR comments. Please respond as soon as you can, and let me know if you have any questions! The GitHub UI can definitely be confusing, so I'm happy to walk you through with screenshots or a live demo screenshare walkthrough in a 1:1 appointment if that's helpful.

File: src/main/webapp/index.html (hunk 1 above)
Comment (CharlesArday) [reply src/main/webapp/index.html:38]: For now I think its good.

File: src/main/webapp/css/main.css [hunk 4]
```diff
@@ -30,3 +30,9 @@ nav li {
   display: inline-block;
   margin-right: 10px
 }
+
+/* Style the team names */
+/* TODO(kyra): change this style */
+.team-names {
+  color: magenta;
```
Comment (hengelman) [inline src/main/webapp/css/main.css:37]: purple is my favorite color :)

File: src/main/webapp/index.html (hunk 1 above)
Comment (kyrabusser) [reply src/main/webapp/index.html:38]: Thanks, Charles--congrats on being our team's first code reviewer!! :tada: I'll resolve this conversation for now. :+1:

**Review by hengelman — APPROVED** [review]

File: src/main/webapp/css/main.css (hunk 4 above)
Comment (kyrabusser) [reply src/main/webapp/css/main.css:37]: :purple_heart: :smile: :purple_heart: 

Do you think we should leave this as magenta, or change to a darker purple?

Comment (hengelman) [reply src/main/webapp/css/main.css:37]: we can leave it for now!

Comment (kyrabusser) [reply src/main/webapp/css/main.css:37]: ok, looking forward to future iterations on this first version!

File: src/main/webapp/index.html (hunk 3 above)
Comment (stanleynwakamma) [reply src/main/webapp/index.html:40]: Great! This whole thing is making much sense to me now 👍

**Pull request merged**